nicsdru_nidirect_theme.theme

```diff
@@ -51,23 +51,35 @@ function nicsdru_nidirect_theme_preprocess_page(array &$variables) {
   // Process search view pages.
   $viewpage = Drupal::routeMatch()->getParameter('view_id');
   if (isset($viewpage)) {
+    $search_layout_classes = ['grid', 'layout--side-n-main', 'search-content'];
     switch ($viewpage) {
-      case "article":
       case "gp_practices":
       case "health_conditions":
       case "publications":
-        $variables['main_content_attr']['class'][] = 'grid';
-        $variables['main_content_attr']['class'][] = 'layout--side-n-main';
-        $variables['main_content_attr']['class'][] = 'search-content';
-        $variables['sidebar_first_attr']['class'][] = 'ga-aside';
-        $variables['sidebar_first_attr']['role'] = 'search';
-        $variables['sidebar_first_attr']['aria-label'] = t('filter search results');
+        $variables['main_content_attr']->addClass($search_layout_classes);
+        $variables['sidebar_first_attr']
+          ->addClass('ga-aside')
+          ->setAttribute('role', 'search')
+          ->setAttribute('aria-label', t('filter search results'));
         break;
     }
   }
 
+  // Contacts page uses multiple routes defined in nidirect_contacts module.
+  $route = Drupal::routeMatch()->getRouteName();
+  if ($route == "nidirect_contacts.default" or $route == "nidirect_contacts.letter") {
```

Comment:
It's standard to use the double pipe operator `||` for OR rather than the keyword `or`. Functionally they achieve the same thing, but they carry different precedence when parsing so could easily introduce confusion in future on top of inconsistencies with core/contrib code.

See https://www.php.net/manual/en/language.operators.logical.php for further info about precedence orders